diff --git "a/C:\\Users\\kesavan\\AppData\\Local\\Temp\\TortoiseGit\\qpdf-25e3a85.000.cpp" "b/D:\\SfQtdevfresh\\SfQtWebKit\\qtbase\\src\\gui\\painting\\qpdf.cpp"
index 3aadd81d6..bca14b6ed 100644
--- "a/C:\\Users\\kesavan\\AppData\\Local\\Temp\\TortoiseGit\\qpdf-25e3a85.000.cpp"
+++ "b/D:\\SfQtdevfresh\\SfQtWebKit\\qtbase\\src\\gui\\painting\\qpdf.cpp"
@@ -1282,6 +1282,14 @@ void QPdfEngine::drawTextItem(const QPointF &p, const QTextItem &textItem)
   QString m11Val =  QString::number(d->stroker.matrix.m11());
   std::string m11Valstring = m11Val.toStdString();
   const char * m11Valchar = m11Valstring.c_str();
+  std::string factor = m11Valchar;
+
+  QString m12Val =  QString::number(d->stroker.matrix.m12());
+  std::string m12Valstring = m12Val.toStdString();
+  const char * m12Valchar = m12Valstring.c_str();
+
+  if(factor == "0")
+     m11Valchar = m12Valchar;
 
   QString dxVal =  QString::number(d->stroker.matrix.dx());
   std::string dxValstring = dxVal.toStdString();
